pkg/cloudprovider/azure.go

```diff
@@ -128,6 +132,7 @@ func (a *Azure) AssignPrivateIP(ip net.IP, node *corev1.Node) error {
 			Subnet:                          (*networkInterface.IPConfigurations)[0].Subnet,
 			Primary:                         &untrue,
 			LoadBalancerBackendAddressPools: (*networkInterface.IPConfigurations)[0].LoadBalancerBackendAddressPools,
+			ApplicationSecurityGroups:       &applicationSecurityGroups,
```

Comment:
Can we just  do:
```
ApplicationSecurityGroups: (*intf.IPConfigurations)[0].InterfaceIPConfigurationPropertiesFormat.ApplicationSecurityGroups
```
instead of introducing `getApplicationSecurityGroups`?

Reply:
yes, much cleaner.